homeassistant/components/stookalert/binary_sensor.py

```diff
@@ -0,0 +1,76 @@
+"""This component provides support for Stookalert Binary Sensor."""
+import logging
+import voluptuous as vol
+import stookalert
+from datetime import timedelta
+
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.components.binary_sensor import PLATFORM_SCHEMA
+from homeassistant.const import CONF_NAME, STATE_OFF, STATE_ON
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(minutes=60)
+CONF_PROVINCE = "province"
+DEFAULT_NAME = "Stookalert"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+		vol.Required(CONF_PROVINCE): vol.In([
+            "drenthe", "flevoland", "friesland", "gelderland", "groningen", "limburg", 
+            "noord-brabant", "noord-holland", "overijssel", "utrecht", "zeeland", "zuid-holland"]),
```

Comment:
Please move this list to a constant and then use that constant. E.g., `vol.In([PROVINCES])`

I suggest to write a custom validator and use `stookalert.PROVINCES`. Otherwise you have to maintain the list of supported provinces at two places.

Reply:
Changed to the first comment. Probably I'll drop the validation in the PyPi component, because the API will only supply one of those provinces.